### ISSUE TITLE
Add Appimage build

### ISSUE THREAD

**Issue by SebastianMpl** [mgoltzsche/podman-static#97]
Hi there,

It would be great if you could add an Appimage build for Linux x86_64.
Currently your release requires using sudo to move binaries to certain system directories.
It works fine, although I'm thinking about a way that won't require root.

Thank you,

**Comment by mgoltzsche** [comment]
Sounds like a great idea btw. I didn't know about AppImage until you mentioned it - very interesting.
I didn't find the time to look into that yet and I won't have the time soon.
Thus, please feel free to create a pull request.
Otherwise I might come back to this but it may take some time.

**Comment by cryi** [comment]
I do not believe this is correct. You do not need root to install podman. Appimage won't provide much benefit. `podman` here is statically linked. Just drop the binaries in places you like. Only issue may be the `quadlet` which expects podman in specific path. But except that it is all about placing podman and friends whenever you like and configure podman through configuration files to find crun and others.

**Comment by mgoltzsche** [comment]
Right, podman can be installed by a non-root user already but not exactly the way it is documented within the readme which instructs users to extract the tar to paths on the host that are owned by root. Thus, it would be a matter of convenience to simplify the installation for non-root users and I am in favour of doing that - I just still don't have time for that now but I'd be happy if somebody else would document that within the readme and/or even prepare an AppImage build which seems to potentially simplify the installation for those users.

Fwiw somebody who already installed podman as non-root user shared in #89 how he did that.

**Comment by tobwen** [comment]
Don't forget this please: https://github.com/popsUlfr/podman-appimage
